app/services/userService.js

```diff
@@ -22,6 +24,24 @@ exports.findUserByEmail = email => UserModel.findOne({ where: { email: formatEma
 
 exports.findUserById = id => UserModel.findByPk(id);
 
+exports.findTopWeetAuthor = async () => {
+  try {
+    const mostWordWeetAuthor = await WeetModel.findAll({
+      limit: 1,
+      where: {},
+      order: [[sequelize.fn('length', sequelize.col('content')), 'DESC']]
+    });
+
+    let topUser = null;
+    if (mostWordWeetAuthor[0]) topUser = await UserModel.findByPk(mostWordWeetAuthor[0].userId);
+
+    return topUser;
```

Comment:
You are returning null anyway. You should throw an error if it is not found, and then catch it if you do not want it to reach the client (like this case, where the job should not throw errors)